src/openforms/registrations/contrib/json_dump/config.py

```diff
@@ -47,6 +47,32 @@ class JSONDumpOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serialize
         required=True,
         min_length=1,
     )
+    fixed_metadata_variables = serializers.ListField(
+        child=FormioVariableKeyField(),
+        default=[
+            "public_reference",
+            "form_name",
+            "form_version",
+            "form_id",
+            "registration_timestamp",
+            "auth_type",
+        ],
+        label=_("Fixed metadata variable key list"),
+        help_text=_(
+            "A list of required variables to use in the metadata. These include "
+            "the registration variables of the JSON dump plugin"
+        ),
+        required=False,
+    )
+    # TODO: this list should be validated against the available FormVariable records
+    #  that exist for the form.
+    additional_metadata_variables = serializers.ListField(
+        child=FormioVariableKeyField(),
+        label=_("Additional metadata variable key list"),
+        help_text=_("A list of additional variables to use in the metadata"),
+        required=False,
+        default=[],
```

Comment:
```suggestion
        default=list,
```
don't know if this will work, I think so. Mutable defaults are tricky in Python and I prefer to avoid them entirely if possible.

Reply:
Yeah it even says in the documentation that this is the preferred way. Didn't think about that it could be a factor here..